### DISCUSSION TITLE
New module with iframe

### DISCUSSION THREAD

**Discussion by adirtp** [YetiForceCompany/YetiForceCRM#14286] [How to]
Hi there!
It seems to be simple, but i cant find the way or info about it.
I want to create a new module/page inside yeti, with iframe on it, or just a basic php file that i can edit.

After creating a new module , how can i change the table template with own code? or thats not the way to do that?

Thanks you guys.

**Comment by apcloic** [comment]
@ad1rie1 
Hi,
Maybe you could you give more details and tell us what you want to do with your iframe ?

**Reply by adirtp** [reply]
Thanks for your answer.
just want to display it on the body page, still using the yeti template of header, footer & side menu - but to be able to replace the table layout to a customized one.

Im using "rocket.chat" , a self hosted chat with an option to integrate it on any page using iframe.
or maybe just to create a blank PHP template for specific module (instead of the table DB layout) so it will be easier to edit the code and display the content

thanks!

**Reply by apcloic** [reply]
It looks like you need to create an extension module and a view :
https://gitstable.yetiforce.com/index.php?module=ModuleManager&parent=Settings&view=List

See also the following thread :
#3967

**Reply by adirtp** [reply]
Yes. i saw that discussion but cant get through.
the only clue there is "You need to create an extension type module and create one view" which i dont fully understand.
I created an extension module. can you please tell me whats the next steps?
I really appreciate the time & help mate.

**Reply by apcloic** [reply]
@adirtp bear in mind that you need programming skills, there is no ready-made solution for that.
You may try to check the code of the following "Our sites" module which was removed since YF v6, this old module was used to display urls inside YF :
https://github.com/YetiForceCompany/YetiForceCRM/tree/5.3.0/modules/Portal

I dont' know if rocket.chat will work this way but you could simply try to install this old module and test.

Don't forget to backup everything before testing.
